doc/how_to/authentication/basic.md

```diff
@@ -45,32 +69,4 @@ When serving an application dynamically using `pn.serve` you can also provide a
 For a more in-depth understanding of custom template take a look at the [how to > Build a Custom Template](../templates/template_custom.md) guide.
 ```
 
-If you want to customize the authentication template you can provide a custom template with the `--basic-login-template` CLI argument. The template needs to submit `username` and `password` to the `/login` endpoint of the Panel server, e.g. the form of the default template looks like this:
-
-```html
-<form class="login-form" action="/login" method="post">
-    <div class="form-header">
-        <h3><img src="https://panel.holoviz.org/_images/logo_stacked.png" width="150" height="120"></h3>
-        <p> Login to access your application</p>
-    </div>
-    <div class="form-group">
-        <span style="color:rgb(255, 0, 0);font-weight:bold" class="errormessage">{{errormessage}}</span>
-    </div>
-    <p></p>
-    <!--Email Input-->
-    <div class="form-group">
-        <input name="username" type="text" class="form-input" autocapitalize="off" autocorrect="off" placeholder="username">
-    </div>
-    <!--Password Input-->
-    <div class="form-group">
-        <input name="password" type="password" class="form-input" placeholder="password">
-    </div>
-    <!--Login Button-->
-    <div class="form-group">
-        <button class="form-button" type="submit">Login</button>
-    </div>
-    <div><small>
-    <p><a href="https://panel.holoviz.org/how_to/authentication/index.html">See the documentation</a> for a full discussion.</p>
-    </small></div>
-</form>
-```
+If you want to customize the authentication template you can provide a custom template with the `--basic-login-template` CLI argument. The template needs to submit `username` and `password` to the `/login` endpoint of the Panel server, e.g. the form of the default template looks like this. Check out the default template [here](https://github.com/holoviz/panel/blob/main/panel/_templates/basic_login.html) for inspiration.
```

Comment:
It helps me more to see the entire template than a part of it. And it also requires work to keep the subset here updated. So replaced it with a link instead.